src/app/services/sms/sms.service.ts

```diff
@@ -237,6 +253,8 @@ const sendSms = (
         )
       }
 
+      ddClient.increment('sms.send', 1, 1, ddTags)
```

Comment:
did we abandon using winston as a transport? If this PR not using winston/centralized logging is just due to the urgent nature, we may want to add a TODO here so it's more obvious

Reply:
This is not logging, it's metric tracking. the parent ddclient is the centralized statsd tracker